src/styles/components/_card.scss

```diff
@@ -101,12 +121,47 @@
   }
 }
 
+@mixin bsds-modifier-error-text {
+  @include font-family-base-regular;
+  @include font-size($fs-base);
+  color: $feedback-error-40;
+  background-image: url("../../assets/images/icon-error.svg");
+  background-size: space(2x);
+  background-repeat: no-repeat;
+  background-position: left;
+  padding: 0 0 0 space(3x);
+}
+
 .bsds-card-with-image
 {
   .bsds-card-title
   {
     margin-top: space(md);
   }
+
+  .bsds-icon-8x
+  {
+    display: none;
+
+    ~ .bsds-card-content
+    {
+      &::before
+      {
+        @include bsds-modifier-error-text;
+        content: "Image and icon modifiers cannot be combined.";
+      }
+    }
+
+    ~ .bsds-card-title
+    {
+      &::before
+      {
+        @include bsds-modifier-error-text;
+        content: "Image and icon modifiers cannot be combined.\A";
+        white-space: pre-wrap;
```

Comment:
We can cut this if we add display: block to the ::before, which will also let us match line wrapping for the error messages